01/app.js

```diff
@@ -1,2 +1,12 @@
 const randomNumber = Math.round(Math.random() * 20);
-console.log(randomNumber);
+document.write('Wylosowana liczba to: ' + randomNumber + '<br>');
+
+if (randomNumber > 5) {
+    document.write('Liczby od 5 do wylosowanej wartości to:');
+    for (let i = 5; i <= randomNumber; i++) {
+        // działa bez deklaracji let dla i ale rozumiem, że deklarowanie zminnej jest dobra praktyką??
```

Comment:
Nawet trzeba! Jeśli nie napiszesz `let` to będzie tworzona zmienna globalna - a tak nie powinno być :)